app/src/main/AndroidManifest.xml

```diff
@@ -152,9 +161,22 @@
         <category android:name="android.intent.category.DEFAULT"/>
         <category android:name="android.intent.category.BROWSABLE"/>
       </intent-filter>
+      <intent-filter>
+        <action android:name="android.intent.action.VIEW"/>
```

Comment:
As far as I know, ParseDeepLinkActivity is related to Google+ share. 
I think we should move this intent-filter to `EventActivity`. What do you think?